templates/createrepo-update.sh.erb

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+<%-
+    command = @createrepo_update
+-%>
+
+if [ "$(whoami)" != '<%= @repo_owner %>' ]; then
+    echo "You really should be '<%= @repo_owner %>' while running '$0'"
+    exit 1
+fi
+
+<%- if @use_filter -%>
+[ $# -eq 0 ] || echo "$1" | grep -q '\.rpm$' || exit
```

Comment:
This filter thing seems a bit organizationally specific. Nowhere else in the script is there any usage of a `.rpm` argument. When reading it I found myself going back and forth looking for hints of some extra usage but found none. I can see this type of functionality could be useful, although I generally call the update script after uploading new `.rpm`s which render this moot and at worst an extra 10 sec delay in a deployment pipeline.

I'm torn between these options:
a) stop depending on a string ending with `.rpm` being a trigger for this and changing it to a parameter of a reasonable name.
b) not introducing this and adding a `$update_script_template` parameter to the module so that anyone can override the default script template with custom logic.

I think we should do `b)` whichever way we go. What do you think?

Reply:
We use incrond to trigger the update script whenever a new file arrives in the repository directory. In this call, incrond passes the filename as argument, hence we use it to check whether the script was called with an rpm-file as argument. I agree that this seems a bit overcomplicating. 

I propose to should do remove the use_filter functionality altogether, as it is more logical to use a wrapper script with incrond if needed, that does these checks.